MUXSDKStats/MUXSDKStats/MUXSDKConnection.m

```diff
@@ -0,0 +1,57 @@
+#import "MUXSDKConnection.h"
+#import <SystemConfiguration/SystemConfiguration.h>
+
+/*
+    dylanjhaveri
+    Right now this class has 1 significant shortcoming and it is that on
+    a wired ethernet TVOS app it will detect "wifi" as the connection type
+
+    If we can bump the target iOS version for this SDK all the way up to iOS12
```

Comment:
You may have explained this already and I missed it but was there a reason we couldn't check if iOS12 was available and conditionally use `NWPathMonitor`?

Reply:
It's because of the linker when we run `xcodebuild` 😢 

As I understand it, because we're targeting ios 9+, when we run xcodebuild the linker was not building when it saw `nw_path_monitor_create` and things that are only in ios 12+ (even after I had that code surround by obj-c preprocessor macros).

If there is a way to include ios 12+ functionality while still building our SDK for ios 9+, I was unable to do it. Maybe we can with XCFrameworks? Worth trying when we make that change, at least.